### ISSUE TITLE
Wrong index in Fig on page 384?

### ISSUE THREAD

**Issue by olivertomic** [rasbt/python-machine-learning-book-2nd-edition#76]
Hi Sebastian. 

On page 384 in the MLP figure there may be a typo? In the 2nd layer for the last activation unit I would have expected the sub-index to "d", not "h". That would fit also with the (m x d) dimension of W(h) as mentioned on page 389 (first section).

**Comment by rasbt** [comment]
Good catch, you are right! I am just fixing this for the 3rd edition. Many thanks!

**Comment by olivertomic** [comment]
No worries, I am glad I could help. When should we expect the third edition to be published?

**Comment by rasbt** [comment]
We are currently aiming for ~mid October. Vahid and I have been working on a bunch of new content (mainly rewriting everything after Ch 12 wrt Tf 2.0 and adding a GAN and reinforcement learning chapter) -- currently, we are going over it a few more times to make sure everything is in good shape. Shouldn't be too long from now on.

**Comment by olivertomic** [comment]
Excellent! Looking forward to it. Keep on the good work!
